packages/ember-routing/lib/system/router.js

```diff
@@ -1492,23 +1493,28 @@ function appendLiveRoute(liveRoutes, defaultParentState, renderOptions) {
     set(target.outlets, renderOptions.outlet, myState);
   } else {
     if (renderOptions.into) {
-      deprecate(
-        `Rendering into a {{render}} helper that resolves to an {{outlet}} is deprecated.`,
-        false,
-        {
-          id: 'ember-routing.top-level-render-helper',
-          until: '3.0.0',
-          url: 'https://emberjs.com/deprecations/v2.x/#toc_rendering-into-a-render-helper-that-resolves-to-an-outlet'
-        }
-      );
+      let orphanedOutletsDeprecation = `Rendering into a {{render}} helper that resolves to an {{outlet}} is deprecated.`;
+      if (ENV._ENABLE_ORPHANED_OUTLETS_SUPPORT !== true) {
+        assert(orphanedOutletsDeprecation);
```

Comment:
Is it reasonable to reuse the deprecation warning in the assertion?

Reply:
We should change it from `is deprecated` to`is no longer supported` in the assert case. This does bring up an interesting question as the `{{render}}` helper is going away completely so I don't know if this case will ever be hit post 3.0.